wmt_shields/common/shield_maker.py

```diff
@@ -60,7 +62,11 @@ def dimensions(self):
     def find_resource(self, subdir, filename):
```

Comment:
I would prefer a separate function `find_url_resource` here to make it explicit that an external source is needed.